dynamic-demo-plugin/src/components/ListPage.tsx

```diff
@@ -13,6 +13,7 @@ import {
   RowProps,
   ResourceLink,
   TableColumn,
+  useK8sWatchResources,
```

Comment:
We may need update it to `useK8sWatchResource`

Reply:
done